content/docs/integrations/dynatrace/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Dynatrace
+publishedDate: '2024-04-04T21:00:00.0Z'
+description: How setup Roadie to display data from Dynatrace
+
+humanName: Dynatrace
+logoImage: '../../../assets/logos/dynatrace/Dynatrace_Logo_color_positive_vertical.png'
+integrationType: OSS plugin
+---
+
+## Introduction
+
+With this plugin, you can embed information about Dynatrace problems into your entity pages within Roadie. 
+
+## Step 1 - Obtain Dynatrace API key
+
+You need a Dynatrace API access token to be able to use the Dynatrace plugin. You can follow the instructions on how to retrieve the token on Dynatrace documentation: https://docs.dynatrace.com/docs/dynatrace-api/basics/dynatrace-api-authentication. 
+
+The plugin requires an access token with the following permissions to be able to retrieve relevant information from the Dynatrace REST API:
+* `entities.read`
+* `problems.read`
+* `DataExport` and/or `ExternalSyntheticIntegration` and/or `ReadSyntheticData`
```

Comment:
The reader is probably wondering why they would want one or other of these.

If it was me, I would give it the minimum and then start adding them in to see what changes. But that is a slow process.

Is it possible to elaborate as to what they enabled. Maybe a table with "Permission", "Required/Optional", and "Description of what the permission enables in the plugin".

Reply:
I don't know what these enable. The plugin docs give no hint of these